ansible_collections/arista/avd/molecule/eos_designs_unit_tests/inventory/group_vars/DC1_LEAF_TG_TEST_1.yml

```diff
@@ -0,0 +1,2 @@
+---
+enable_trunk_groups: true
```

Comment:
I have not implemented support for setting this under node_groups. If you think it makes sense to do it there instead of globally, we can discuss that. In my opinion this is one of the common fabric-wide settings you set once for a deployment.